<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="leHost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>www.naver.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect(HTTP)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CKssl">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>12</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SSL(HTTPS)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>771</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>681</width>
     <height>201</height>
    </rect>
   </property>
   <property name="plainText">
    <string>GET / HTTP/1.1
Host: www.naver.com

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSend">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
